fluidBuild.config.cjs

```diff
@@ -440,7 +439,6 @@ module.exports = {
 				// internal partners or internal CI requirements.
 				internalFeed: [
 					// TODO: We may not need to publish test packages to the internal feed, remove these exceptions if possible.
```

Comment:
Remove this TODO?

Reply:
It's still relevant for `@fluid-internal/test-service-load` which remains.